src/core/operations/SetOperations.js

```diff
@@ -0,0 +1,202 @@
+import Utils from "../Utils.js";
+
+/**
+ * Set operations.
```

Comment:
I'd probably rename this file to `SetOps.js` to fit with the convention (e.g. `BitwiseOps.js`)